presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -45,3 +45,89 @@ which is configured with a carefully selected set of options. This behavior
 name of a different program such as ``more`` or `pspg <https://github.com/okbob/pspg>`_,
 or it can be set to an empty value to completely disable pagination.
+
+Output Formats
+--------------
+
+The Presto CLI provides the option ``--output-format`` to control how the output is
+displayed when running in noninteractive mode. The available options shown in the
+following table must be entered in uppercase. The default value is ``CSV``.
+
++---------------------+------------------------------------------------------------+
+| Option              | Description                                                |
++=============+=======+============================================================+
+| CSV                 | Comma-separated values, each value quoted. No header row.  |
++---------------------+------------------------------------------------------------+
+| CSV_HEADER          | Comma-separated values, quoted with header row.            |
++---------------------+------------------------------------------------------------+
+| CSV_UNQUOTED        | Comma-separated values without quotes.                     |
++---------------------+------------------------------------------------------------+
+| CSV_HEADER_UNQUOTED | Comma-separated values with header row but no quotes.      |
++---------------------+------------------------------------------------------------+
+| TSV                 | Tab-separated values.                                      |
++---------------------+------------------------------------------------------------+
+| TSV_HEADER          | Tab-separated values with header row.                      |
++---------------------+------------------------------------------------------------+
+| JSON                | Output rows emitted as JSON objects with name-value pairs. |
++---------------------+------------------------------------------------------------+
+| ALIGNED             | Output emitted as an ASCII character table with right-     |
+|                     | aligned numeric values and left-aligned string values.     |
++---------------------+------------------------------------------------------------+
+| VERTICAL            | Output emitted as record-oriented top-down lines, one per  |
+|                     | value.                                                     |
++---------------------+------------------------------------------------------------+
+| NULL                | Suppresses normal query results. This can be useful during |
+|                     | development to test a query's shell return code or to see  |
+|                     | whether it results in error messages.                      |
++---------------------+------------------------------------------------------------+
+
+Examples
+^^^^^^^^
+
+Consider the following command run as shown, or with ``--output-format CSV``::
+
+   presto --execute 'SELECT nationkey, name, regionkey FROM tpch.sf1.nation LIMIT 3'
+
+The output is::
+
+  "0","ALGERIA","0"
```

Comment:
:: in our setup implies code style "sql" .. so in the exampes here pretty much always need to use

.. code-block:: none

Reply:
OK, coming up. Except it was like pulling teeth to get the JSON output right, and it is actual JSON. Leave that highlighting in, I hope and assume?

Reply:
Et voila.

Reply:
Looks good now .. and yes .. json works, also properties, shell, java ... but you will learn more about that stuff soon enough ;-)